sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -55,6 +55,7 @@
     'Top',
     'ToDict',
     'ToList',
+    'ToString',
```

Comment:
ToString is not a combiner. Could you move the changes to transforms/util.py (similarly tests to util_test.py)

Reply:
sure.